doc/api/http.md

```diff
@@ -3249,7 +3249,7 @@ changes:
   * `highWaterMark` {number} Optionally overrides all `socket`s'
     `readableHighWaterMark` and `writableHighWaterMark`. This affects
     `highWaterMark` property of both `IncomingMessage` and `ServerResponse`.
-    **Default:** See [`stream.getDefaultHighWaterMark()`][].
+    **Default:** `undefined`. Sockets will use the default watermark value at the time the request arrives. See [`stream.getDefaultHighWaterMark()`][].
```

Comment:
I would keep it consistent and add a line break after 80 characters.

Reply:
The lint job fails for this reason.

Reply:
Addressed